configs/js.json

```diff
@@ -19,6 +19,7 @@
     { "target": "alanshaw/it-tar" },
     { "target": "alanshaw/it-to-stream" },
     { "target": "alanshaw/it-ws" },
+    { "target": "ipfs-shipyard/ipfs-geoip" },
```

Comment:
I think for this to work, we need `npm-service-account-ipfs` to have write access to https://www.npmjs.com/package/ipfs-geoip  

@protocol/ipdx  I've invited `npm-service-account-ipfs` to that project, are you able to approve NPM invitation? Or is this happening automatically on the first run?

Reply:
I just noticed, the repo is in `ipfs-shipyard` org, so we should use `npm-service-account-ipfs-shipyard` account here. We will also need to add the NPM token as a secret to the repo. I'm going to do both tomorrow.

Reply:
@galargh should I wait to merge this or can you please merge this PR when you're good?

Reply:
I changed the account that has access to the repo from `ipfs` one to the `ipfs-shipyard` one and updated the `NPM_TOKEN` in the repo accordingly.

Reply:
The release made it to the repo, see https://github.com/ipfs-shipyard/ipfs-geoip/pull/102

Reply:
I think it'll need some fine-tuning before the tests pass.